package.json

```diff
@@ -19,7 +19,7 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "git clean -fdx -e node_modules && yarn compile",
```

Comment:
Should this stay in? The release of 2.2.0 had issues with stale files in the lib folder.
Alternatively one could add a clean step in rollup.

Reply:
agree, this should stay in, fixing it now

Reply:
c3228873596a81973eb8307fc2e2f7397d36465b